q2_diversity_lib/tests/test_alpha.py

```diff
@@ -16,38 +17,190 @@
 import pandas as pd
 import pandas.util.testing as pdt
 
+import copy
 
-class AlphaTests(TestPluginBase):
+nonphylogenetic_measures = [observed_features, pielou_evenness,
+                            shannon_entropy]
+
+
+class GeneralTests(TestPluginBase):
+    package = 'q2_diversity_lib.tests'
+
+    def setUp(self):
+        super().setUp()
```

Comment:
`10001010001010 <in the mainframe>`

Reply:
huh? :confused:

Reply:
You're in the mainframe. Duh.

Reply:
Ask @ebolyen. He lives in the mainframe.

Reply:
(he is the mainframe (?))

Reply:
you can't make me

Reply:
I shan't make yea.

@ChrisKeefe --- 1010000111010101 && references to being in the mainframe usually are code for "that is some fancy hacking!" I thought it was fancy, b/c personally, I always forgot to `super()` my test setups --- I was happy to see you had remembered! You're an elite hacker now (in the mainframe).

Reply:
:man_technologist:

Reply:
l33tz0rs! :man_technologist: